### ISSUE TITLE
Feature Request: Suppresive Fire module

### ISSUE THREAD

**Issue by neilzar** [zen-mod/ZEN#222]
**Is your feature request related to a problem?**
There is no way to make AI shoot at a building/area without any enemy targets in sight.

**Solution you'd like:**
Similar to Achilles, a module that can be placed on a group or unit making that group or unit lay down suppressive fire on a selected target.

**Comment by bux** [comment]
For reference:

https://github.com/acemod/ACE3/blob/master/addons/zeus/CfgVehicles.hpp#L214-L219
https://github.com/acemod/ACE3/blob/master/addons/zeus/functions/fnc_moduleSuppressiveFire.sqf

**Comment by CreepPork** [comment]
Achilles’ implementation allows more config options but I also like the simplicity of ACE’s. I’d prefer some kind of balance between them or maybe two modules all together or ACE’s tied to a keybind and then only Achilles’ is found in the module tree.

I dunno, should be open for suggestions.

**Comment by neilzar** [comment]
I forgot the ACE one existed 😄 

I don't care much if we created one in Zen, but there are probably some people that would like it.

**Comment by bux** [comment]
Neveer used Achilles before, but how well does it work with long range engagements (over +600m) and can it be used for a long sustained suppressive fire?

In one of our last Zeus missions I struggled to have multiple units (5-6) suppress our forces in a long range engagement over multiple minutes.

ACE one is struggeling with flora and make the suppressive fire hit a bush or tree that was in the way.

**Comment by BlackSeaSurfer** [comment]
A Suppression Module similar or exactly the same to Achilles would be awesome, The ace one its kinda of wonky, not having very much customization. I'd love to be able to change duration, so that the enemy isn't firing for too long or is keeping a steady burst of fire.

**Comment by ValrathCommander** [comment]
Suppressing with ACE doesn't work half of the time, with the desired 'suppression' being a few shots before refusing to fire. Would be great if ZEN implemented a suppression feature like what Achilles did.

**Comment by EllmansWorld** [comment]
I'd love to see Achillies old suppression module be implemented into ZEN since the ace3 one isn't reliable.
+ the Achillies one would ensure that the AI didn't fire suppressive fire into a object.

**Comment by rpgkayak** [comment]
My unit made the transition from Achilles to ZEN recently, and I found it a very worthy successor. The one module that I desperately missed, however, was Suppressive Fire. I've used the ACE module numerous times, but I'm still not sure exactly how it works. The fire rate and duration seem variable and sporadic. The Achilles module allowed the ability to set these features, so it would run as long or as short as you needed, and the units would fire irrespective of distance or obstacles. It was probably the single best way to keep players' heads down and I would love to see it added to ZEN.
